docs/pages/blog/mui-x-v7.md

```diff
@@ -296,7 +296,7 @@ This feature greatly improves the experience for screen reader users, making int
 <DatePicker enableAccessibleFieldDOMStructure />
 ```
 
-For more details, please check out the [documentation section](/x/react-date-pickers/fields/#accessible-dom-structure).
```

Comment:
This link seems to exist, what's wrong with it? https://mui.com/x/react-date-pickers/fields/#accessible-dom-structure

Reply:
Huh, that is strange. It's reported as broken in https://github.com/mui/mui-x/pull/15735 and causing that check to fail. Maybe something's going wrong with `pnpm docs:link-check` in the X repo?

Reply:
Never mind, it appears to have been something strange happening only on my MUI X fork. I'll close this now.